pysmarthashtag/api/utils.py

```
@@ -35,9 +40,21 @@ def _create_sign(nonce: str, params: dict, timestamp: str, method: str, url: str
 
 
 def generate_default_header(
-    device_id: str, access_token: str, params: dict, method: str, url: str, body=None
+    device_id: str,
+    access_token: str,
+    params: dict,
+    method: str,
+    url: str,
+    body=None,
+    vin: str = None,
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)